google/cloud/storage/ci/run_integration_tests_emulator_bazel.sh

```diff
@@ -30,7 +30,14 @@ shift
 BAZEL_VERB="$1"
 shift
 
-bazel_test_args=()
+EMULATOR_SHA="$(find "${HOME}/.local/" -type f -name '*.py' -print0 |
```

Comment:
Q.  Is there a sub-directory that might limit this to just emulator files?

Reply:
Not a subdirectory, but we can find the list of files with some python stuff. PTAL.